Vates/VatesAPI/src/vtkMDHexFactory.cpp

```diff
@@ -120,8 +117,7 @@ void vtkMDHexFactory::doCreate(
       API::IMDNode *box = boxes[i];
       Mantid::signal_t signal_normalized = (box->*normFunction)();
 
-      if (std::isfinite(signal_normalized) &&
-          m_thresholdRange->inRange(signal_normalized)) {
```

Comment:
@quantumsteve Do you have any idea what the runtime improvement is as a result of removing this?